src/views/EdgeApplicationsDeviceGroups/Drawer/index.vue

```diff
@@ -61,9 +61,10 @@
     emit('onSuccess')
   }
 
-  const handleSuccessCreate = () => {
+  const handleSuccessCreate = (response) => {
+    const createItemId = response.id
     handleTrackCreation()
-    emit('onSuccess')
+    emit('onSuccess', createItemId)
```

Comment:
```suggestion
   handleTrackCreation()
    emit('onSuccess', response)
```
Dessa forma podemos utilizar em outros cenários.